app/javascript/orangelight/bookmark_all.es6

```diff
@@ -30,10 +32,14 @@ export default class BookmarkAllManager {
   }
 
   bookmark_all() {
```

Comment:
The actual changes are in these two methods: `bookmark_all` and `unbookmark_all`